config/initializers/api_v2_methods.rb

```diff
@@ -1,414 +0,0 @@
-# frozen_string_literal: true
-
-GUMROAD_API_V2_METHODS = [
```

Comment:
This is used in an erb file, so this is still needed. Make sure that everything we deleted actually don't occur anywhere else in the codebase (including deletions in other files).